### PR TITLE
Rig fixes

### LINKED ISSUE #655: BUG: No hardsuit interface tab when wearing hardsuit
#### Description of issue

No hardsuit interface tab near Info section of UI.

#### Difference between expected and actual behavior

I would be able to use hardsuit verbs in a hardsuit interface tab, next to IC and Object.
I was not able to do so. I was not able to see a hardsuit interface tab.
#### Steps to reproduce
Equip and activate RIGsuit.
See problem.

#### Specific information for locating

N/A

#### Length of time in which bug has been known to occur

N/A, presumably since rebase

#### Client version, Server revision & Game ID

514, [02fc99c8bedd152fb3817ad05ce4941b072cec10](https://github.com/Foundation-19/Foundation-19/commit/02fc99c8bedd152fb3817ad05ce4941b072cec10) - dev - 2022-08-06, Game ID: cjj-ci5v

#### Issue bingo

- [x] Issue could be reproduced at least once
- [x] Issue could be reproduced by different players
- [x] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/vlggms/tegustation-bay12/issues)

### PR DESCRIPTION
**Pull request by checkraisefold**
## About the Pull Request

what it says on the tin
tested and working
closes #655 

## Why It's Good For The Game

emps bricking rigs was stupid as fuck and probably unintentional, and the tab not being there was straight up a bug with tgui port

## Changelog

:cl:
fix: Rigs are no longer bricked if they're EMPed while turned off.
fix: Hardsuit verbs tab shows up upon putting on a rig without having to refresh tgui.
/:cl:

the method for adding tabs/verbs to the statpanel properly is from the verb helpers in https://github.com/Foundation-19/Foundation-19/blob/dev/code/_helpers/verbs.dm, let me know if you want me to make a new helper for adding verbs to the statpanel without adding them to mob/client verbs since they're already verbs of the object, because people did NOT know the right way in #655

### CODE REVIEW

**Review by MACIEKBAKI — APPROVED** [review]

File: code/modules/clothing/spacesuits/rig/rig.dm [hunk 1]
```diff
@@ -695,6 +700,17 @@
 		wearer.wearing_rig = src
 		update_icon()
 
+		if(M.client)
```
Comment (ivanmixo) [inline code/modules/clothing/spacesuits/rig/rig.dm]: Early returns

File: code/modules/clothing/spacesuits/rig/rig.dm [hunk 2]
```diff
@@ -800,6 +816,16 @@
 	if(wearer)
 		wearer.wearing_rig = null
 		wearer = null
+	if(user.client)
```
Comment (ivanmixo) [inline code/modules/clothing/spacesuits/rig/rig.dm]: Early returns

**Review by ivanmixo — CHANGES_REQUESTED** [review]

File: code/modules/clothing/spacesuits/rig/rig.dm (hunk 1 above)
Comment (checkraisefold) [reply code/modules/clothing/spacesuits/rig/rig.dm]: fixed you picky fuck

**Comment by checkraisefold** [comment]
ready to merge

**Review by ivanmixo — APPROVED** [review]

**Pull request merged**